src/gc/gc.d

```diff
@@ -2033,6 +2208,13 @@ struct Gcx
         return 1;
     }
 
+    static struct ScanRange
+    {
+        void* pbot;
+        void* ptop;
+        Pool* pool;
```

Comment:
I was hoping for any function calls dealing with ScanRange to be inlined, though I didn't check yet.